addons/mail/static/src/components/message_notification_popover_content/message_notification_popover_content.xml

```diff
@@ -11,6 +11,9 @@
                             <t t-if="notification.partner">
                                 <span class="o_MessageNotificationPopoverContent_notificationPartnerName" t-esc="notification.partner.nameOrDisplayName"/>
                             </t>
+                            <t t-elif="notification.unpartnered_email">
+                                <span class="o_MessageNotificationPopoverContent_notificationPartnerName" t-esc="notification.unpartnered_email"/>
```

Comment:
class name should be different as it contains something else than a partner name